File: ClientApp/src/models/mesa/pedidoItem.ts
import { Pedido } from './pedido';

export interface PedidoItem {
  pedidoItemId: number;
  pedidoId: number;
  produtoId: number;
  quantidade: number;
  observacao: string;
  ativo: boolean;
  pedidoPor: number;
  canceladoPor: number;
  pedidoEm: Date;
  preco: number;
  servido: boolean;
  cortesia: boolean;
  pedido: Pedido;
}

File: Models/PedidoItem.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace chamonix.Models
{
    public class PedidoItem
    {
        [Key]
        public int PedidoItemId { get; set; }
        public int PedidoId { get; set; }
        public int ProdutoId { get; set; }
        public byte Quantidade { get; set; }
        public string Observacao { get; set; }
        public bool Ativo { get; set; }
        public int PedidoPor { get; set; }
        public int CanceladoPor { get; set; }
        public DateTime PedidoEm { get; set; }
        public decimal Preco { get; set; }
        public bool Servido { get; set; }
        public bool Cortesia { get; set; }

        public Pedido Pedido { get; set; }        
        public Produto Produto { get; set; }
    }
}